src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int defaultLength = 10;
```

Comment:
```suggestion
    private final static int DEFAULT_LENGTH = 10;
```
Make fields defaultLength and defaultColor constant